documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -324,7 +324,7 @@ Each message written to the schema change topic will have a message key that con
   {
     "schema": {
       "type": "struct",
-      "name": "io.debezium.connector.mysql.SchemaChangeKey",
```

Comment:
I don't think we'd ever replace these type names with something product-specific. It's the same as for Strimzi, where "strimzi" e.g. shows in CRD version names.

Reply:
Wouldn't this substitution change yield `io.Debezium.connector.mysql.SchemaChangeKey` rather than the expected name of `io.debezium.connector.mysql.SchemaChangeKey`?  I noticed this in a few places such as class names and URLs for example where we are substituting `debezium` with `{prodname}` which is defined as `Debezium`.

Reply:
Yes, exactly. This one shouldn't be substituted.